src/Resources/public/javascript/autocomplete.js

```diff
@@ -14,55 +14,37 @@ var ZichtQuicklistAutocomplete = (function($) {
         }
 
         $text
-            .focus(function(){$text.select();})
+            .focus(function () {
+                $text.select();
+            })
             .autocomplete({
                 appendTo: ".zicht-ui-autocompletecontainer_" + $text.attr('id'),
-                source: function(req, resp) {
+                source: function (req, resp) {
                     params.pattern = req.term;
                     $.get(
                         service_url,
                         params,
                         resp
                     );
                 },
-                select: function(e, ui) {
+                select: function (e, ui) {
                     $hidden.val(ui.item.id);
                     $text.val(ui.item.label);
                     callback(ui.item);
+                },
+                change: function(e, ui) {
+                    var value = $(e.target).val();
+                    if (value && new RegExp('^(http|https|mailto):', 'i').test(new String(value))) {
+                        $hidden.val(value);
+                    }
                 }
             })
         ;
         return $text;
     }
 
-    function initCheckboxControl($hidden, $text, $checkbox) {
```

Comment:
Waarom moet dit eruit?

Reply:
Er zat ooit een checkbox in het template, maar niemand begreep de UX hiervan en ik weet ook niet wat dat ding eigenlijk deed. Die is al een tijdje uit het template weg maar de javascript ervan staat er blijkbaar nog